shared/js/project.js

```diff
@@ -125,6 +135,12 @@ var projects = module.exports = {
     }
   },
 
+  showAdmin: function() {
+    if (typeof dashboard.admin.showProjectAdmin === 'function') {
```

Comment:
The check reads like `dashboard.admin` is always expected to exist, but `showProjectAdmin` may be conditionally added.  I'd say remove it, because it will still blow up if `dashboard.admin` doesn't exist.  But no strong preference, totally your call.